manifests/repo.pp

```diff
@@ -3,9 +3,13 @@
 # @api private
 #
 class gitlab_ci_runner::repo (
-  $repo_base_url  = $gitlab_ci_runner::repo_base_url,
-  $repo_keyserver = $gitlab_ci_runner::repo_keyserver,
-  $package_name   = $gitlab_ci_runner::package_name,
+  $repo_base_url        = $gitlab_ci_runner::repo_base_url,
+  $repo_keyserver       = $gitlab_ci_runner::repo_keyserver,
+  $repo_keyid           = $gitlab_ci_runner::repo_keyid,
```

Comment:
$repo_keyid is missing in manifests init.pp

Reply:
Thank you, I removed this parameter, as it was not used.